e2e/testdata/fn-render/subpkg-has-invalid-kptfile/.expected/config.yaml

```diff
@@ -13,3 +13,4 @@
 # limitations under the License.
 
 exitCode: 1
+stdErr: "unable to parse \"Kptfile\": yaml: line 10: mapping values are not allowed in this context"
```

Comment:
morten has PR that changes this message, so you will have to change this after rebase.